sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -71,6 +72,12 @@
 - `DownloadBlobResult::Content-Range` is changed to an `Azure::Core::Http::Range`, an extra field `BlobSize` is added.
 - Remove `Undelete` from `BlobContainerClient`.
 - `BlobRetentionPolicy::Enabled` is renamed to `BlobRetentionPolicy::IsEnabled`, `BlobStaticWebsite::Enabled` is renamed to `BlobStaticWebsite::IsEnabled`.
+- Change type for metadta to case-insensitive `std::map`.
+- Rename `SharedKeyCredential` to `StorageSharedKeyCredential`.
+- Move `StoragePerRetryPolicy` and `StorageSharedKeyPolicy` to `Details` namespace.
```

Comment:
Frame these changelog entries from the perspective of the end user by highlighting what's relevant to them.
```suggestion
- Make `StoragePerRetryPolicy` and `StorageSharedKeyPolicy` private by moving them to the`Details` namespace.
```